datastore.json

```diff
@@ -0,0 +1,28 @@
+{
+  "welcome": {
+    "type": "quick_reply",
+    "text":"Choose a Pill",
+    "options": [
+      "bluePill",
```

Comment:
i would say add one more field as payload in options. Just think what haapens when instead of one or two words we send sentences?

Reply:
Sir elements of the array are identifiers for the elements we want to suggest, which can be mapped to anything we want (text, email, phone). If it is a text then we have corresponding text element defined in the datastore, and we can have sentences inside that element.

Reply:
Let us clear this over a call tomorrow?